src/sentry/models/avatars/user_avatar.py

```diff
@@ -46,5 +50,43 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_useravatar"
 
+    def outboxes_for_update(self, shard_identifier: int | None = None) -> List[ControlOutboxBase]:
+        regions = find_regions_for_user(self.user_id)
+        return [
+            outbox
+            for outbox in OutboxCategory.USER_UPDATE.as_control_outboxes(
+                region_names=regions,
+                shard_identifier=self.user_id,
+                object_identifier=self.user_id,
+            )
```

Comment:
This looks like another list comprehension that could be removed.

Reply:
You are correct